build.rs

```diff
@@ -0,0 +1,116 @@
+use crate::SqlType::{Idempotent, Migration};
+use askama::Template;
+use std::fs::File;
+use std::io::{Error, Read, Write};
+use std::path::{Path, PathBuf};
+use std::string::String;
+use std::{env, fs};
+
+#[derive(Template)]
+#[template(path = "migration-wrapper.sql", escape = "none")]
+struct MigrationWrapper<'a> {
+    filename: &'a str,
+    version: &'a str,
+    body: String,
+}
+
+#[derive(Template)]
+#[template(path = "idempotent-wrapper.sql", escape = "none")]
+struct IdempotentWrapper<'a> {
+    filename: &'a str,
+    body: String,
+}
+
+fn main() {
```

Comment:
I'd add a comment above this to document what this does

Reply:
done